### ISSUE TITLE
SVG <use/> is not rendered

### ISSUE THREAD

**Issue by hakanai** [JetBrains/compose-multiplatform#4102]
**Describe the bug**
If you try to render an SVG which uses the `<use/>` element, the rendering is skipped.

**Affected platforms**
Select one of the platforms below:
- Desktop

**Versions**
- Kotlin version*: 1.9.20
- Compose Multiplatform version*: 1.5.10
- OS version(s)* (required for Desktop and iOS issues): Windows 11
- OS architecture (x86 or arm64): x86
- JDK (for desktop issues): Corretto-17.0.9.8.1

**To Reproduce**
```kotlin
import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.requiredSize
import androidx.compose.material.MaterialTheme
import androidx.compose.material.Surface
import androidx.compose.ui.Modifier
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.unit.dp
import androidx.compose.ui.window.singleWindowApplication

fun main() = singleWindowApplication {
    MaterialTheme {
        Surface {
            Image(
                painter = painterResource("/3C.svg"),
                contentDescription = "3 of Clubs",
                modifier = Modifier.requiredSize(250.dp, 350.dp)
            )
        }
    }
}
```

The SVG: ![3C](https://github.com/JetBrains/compose-multiplatform/assets/43236/6cb30fe8-3cb5-4dad-9141-c06a495488ec)

**Expected behavior**
The image should be rendered as in Chrome, IDEA, and other viewers.

**Screenshots**
![image](https://github.com/JetBrains/compose-multiplatform/assets/43236/df6dd1a0-d7f4-45ea-bf00-e5e6fe169eff)

**Comment by Atlands** [comment]
[For now resources library supports only xml vector drawables and other raster formats (png, jpg. webp)](https://github.com/JetBrains/compose-multiplatform/issues/3713#issuecomment-1733615888)

**Comment by eymar** [comment]
As @Atlands  mentioned, indeed, .svg is not supported as is. You can try to convert your svg into xml. Please follow the comment above.

**Comment by eymar** [comment]
My colleagues told me that actually, .svg is supported for desktop only (without the resources library), not in common. 
So it's a bug in Compose for Desktop or Skiko.

**Comment by eymar** [comment]
Reproduced with 1.6.0-dev1357 too.

**Comment by hakanai** [comment]
Yeah, I've never been sure why Android went and made its own XML format instead of supporting SVG more fully. I remember hitting issues with it way back in the past and thought it might have improved by now.

If I were going to do a rewrite, I think I'd go all the way to rendering the cards in native Kotlin. But that's for later, when I have the time for that kind of exercise. I'll probably make a tool that spits out PDF for printing the actual cards while I'm at it.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4102">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
